package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "zipkin-api",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "typescript": "^5.2.2",
+    "vite": "^5.1.0",
+    "swagger-ui": "^5.11.7",
+    "@types/swagger-ui": "^3.52.4"
```

Comment:
ok so if I understand correctly, instead of unpacking their dist (like we were doing earlier), this is pulling in the UI by its version.

The thing is that there are scripts pushing our assets to the locations they were at before, and unless we setup automation, not sure the value of doing this (as you still have to manually update the content, just by running vite instead of doing it via unzip a dist). what am I missing?

Reply:
Exactly we use their already packaged distributions and don't need to manually copy things around. Yeah you just run `pnpm run build` and the dependencies and the other assets are copied to the dist folder.